modules/simple-payments/simple-payments.php

```diff
@@ -118,6 +155,8 @@ function parse_shortcode( $attrs, $content = false ) {
 		return $this->output_shortcode( $data );
 	}
 
+	function ignore_shortcode() { return; }
```

Comment:
If it's the blog owner (maybe just any admin?) viewing the page, it'd be nice if this could output something like "Simple Payments is not supported by your Jetpack Plan. To learn more, and to upgrade to a supported plan, visit _these resources_. (This message is only shown to site administrators.)"

Reply:
+1 on this!

For example, this is the message I used for the Twitter Timeline widget deprecation:
https://github.com/Automattic/jetpack/blob/758a77efd4c64e9469b2e544cb4ce1c7d7b5ef95/modules/widgets/twitter-timeline.php#L92-L93